src/couch_views/src/couch_views_indexer.erl

```diff
@@ -102,6 +102,8 @@ init() ->
         update_stats => #{}
     },
 
+    process_flag(sensitive, false),
```

Comment:
Why is this needed?

Reply:
Ah derps. That was leftover from testing when trying to trace when aegis was accidentally setting it to true. The other spot has been fixed so this can be removed now.

Reply:
Removed.